# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2023/11
# https://github.com/juglab/cryoCARE_pip/issues/47
# The restriction to ~=1.19 has been introduced due to a conflict with 
# Tensorflow 2.4. However, we are using TF 2.11
diff -ru cryoCARE-0.3.0/setup.py cryoCARE-0.3.0_relax_requirements/setup.py
--- cryoCARE-0.3.0/setup.py	2023-06-15 15:33:40.000000000 +0200
+++ cryoCARE-0.3.0_relax_requirements/setup.py	2023-11-07 11:01:24.464822422 +0100
@@ -23,7 +23,7 @@
     ],
     python_requires='>=3.8',
     install_requires=[
-        "numpy~=1.19.2",
+        "numpy>=1.19.2",
         "mrcfile",
         "csbdeep>=0.7.0,<0.8.0",
         "psutil"
